server/fixtures/crio.conf

```diff
@@ -0,0 +1,43 @@
+[crio]
+root = "/var/lib/containers/storage"
+runroot = "/var/run/containers/storage"
+storage_driver = "overlay2"
+storage_option = ["overlay2.override_kernel_check=1"]
+
+[crio.api]
+listen = "/var/run/crio.sock"
+stream_address = "localhost"
+stream_port = "10010"
+# TODO: this configuration should be on the root but in the example I found it was here
+file_locking = true
```

Comment:
Let's drop this altogether as we're not actually using it

Reply:
I will remove it from the fixture, but it seems that it's being used here: https://github.com/kubernetes-incubator/cri-o/blob/6a43d07bae95b738e910162d581cf791f89427dd/libkpod/container_server.go#L131

Maybe we should create a new issue to remove it from there as well?